frontend/src/components/ProductCard.jsx

```diff
@@ -8,18 +8,13 @@ import { Dialog } from "@chakra-ui/react"
 import { Portal } from '@chakra-ui/react';
 import { CloseButton } from '@chakra-ui/react';
 
-const ProductCard = ({ id, image, name, price, onEditClick }) => {
+const ProductCard = ({ id, image, name, price, onEditClick, onEditSubmit, onDelete }) => {
```

Comment:
very good, use json objects for parameters in typescript function definitions. It's nice for dereferencing when you insert an object as an argument